packages/react-native/Libraries/Utilities/DevMenu.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Since this is technically a new API, we can make this new implementation module private.

- Please relocate to `src/private/`. (I think we can put this under `src/private/utilities/`.)
- It can then reference `src/private/specs/modules/NativeDevMenu.js` directly.
- `packages/react-native/index.js` can then import this module directly.

The net effect is that importing from `'react-native/Libraries/Utilities/DevMenu'` is disallowed (doesn't exist, instead `src/private/` currently signifies this module should not be used directly — it should be imported from `'react-native'`).

(Note that the output from `public-api-test` is incomplete either way this module is wired up 😐. This public/private setup in our codebase is evolving.)

cc @rubennorte